libpod/container.go

```diff
@@ -554,10 +555,19 @@ func (c *Container) Init() (err error) {
 		}
 	}
 
+	// Copy /etc/resolv.conf to the container's rundir
+	runDirResolv, err := c.copyResolvToRunDir()
```

Comment:
You also need to copy the /etc/hosts file.  So it would be better if you make this into a function that takes a path.

Reply:
added